### ISSUE TITLE
Adding "s" char randomly when switch workspace on Ubuntu Jammy LTS

### ISSUE THREAD

**Issue by barayuda** [microsoft/vscode#156419]
Issue Type: <b>Bug</b>

1. Install VSCode on Ubuntu Jammy LTS
2. Open VSCode
3. Tap "New Text File" on File menu
4. Try to switch Workspace
5. VSCode will randomly type "s" char
6. Gif screenshot here: https://drive.google.com/file/d/1nr5LQPKpWaXXGB3ATb9pTHrHK3rg5u7e/view?usp=sharing

VS Code version: Code 1.69.2 (3b889b090b5ad5793f524b5d1d39fda662b96a2a, 2022-07-18T16:14:10.636Z)
OS version: Linux x64 5.15.0-41-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz (8 x 2100)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|0, 0, 0|
|Memory (System)|15.44GB (4.66GB free)|
|Process Argv|--unity-launch|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|wayland|
</details><details><summary>Extensions (36)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.0
vscode-lcov|ale|0.2.0
iconify|ant|0.3.3
vite|ant|0.2.5
vscode-eslint|dba|2.2.6
docs-markdown|doc|0.2.113
docs-preview|doc|0.3.59
gitlens|eam|12.1.2
EditorConfig|Edi|0.16.4
prettier-vscode|esb|9.5.0
shell-format|fox|7.2.2
go|gol|0.35.1
vscode-graphql|Gra|0.6.0
vscode-edit-csv|jan|0.7.1
plantuml|jeb|2.17.3
svg|joc|1.4.19
vscode-gutter-preview|kis|0.30.0
json-table-viewer|len|0.0.5
i18n-ally|lok|2.8.1
git-graph|mhu|1.30.0
dotenv|mik|1.0.1
python|ms-|2022.10.1
vscode-pylance|ms-|2022.7.40
jupyter|ms-|2022.6.1201981810
vetur|oct|0.35.0
polacode|pnp|0.3.4
vscode-commons|red|0.0.6
vscode-yaml|red|1.9.1
LiveServer|rit|5.7.5
lcov-lang|Ser|1.0.1
sass-indented|syl|1.8.22
vscodeintellicode|Vis|1.2.22
vscode-icons|vsc|11.13.0
volar|Vue|0.39.1
vscode-typescript-vue-plugin|Vue|0.39.1
vscode-todo-highlight|way|1.0.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
pylanb8912cf:30529770
vsclangdc:30486549
841h4636:30534993

```

</details>

**Comment by gjsjohnmurray** [comment]
Please use the Start Extension Bisect command to check whether the issue is being caused by one of them.

**Comment by barayuda** [comment]
Thanks for reply @gjsjohnmurray 
I will try, and back to you soon

**Comment by barayuda** [comment]
Hi @gjsjohnmurray cc @alexr00 
New Info regarding this issue:
1. Uninstall VSCode
2. Remove all config
3. Re-install VSCode
4. Make sure no extensions installed
5. Try reproduce issue by step above

Preview:
![image](https://user-images.githubusercontent.com/520463/181678058-b687fe88-5a94-4e49-8ccb-e600f706b509.png)

Screenvideo:
https://drive.google.com/file/d/1GOelKk86H5Ji3ds5QT2AZqUM8HxkHvLk/view?usp=sharing

I still facing same issue, like I mentioned before.
Hopefully can be fix on next release

Thanks in advance

**Comment by alexr00** [comment]
Can you please past your gif directly into the issue?

**Comment by barayuda** [comment]
Hi @alexr00 this the screenshot (gif)
![auto-typing-s-when-switch-workspace-ubuntu](https://user-images.githubusercontent.com/520463/182109817-9093d571-0785-4bd4-90b5-565cb28c7e86.gif)

thanks in advance

**Comment by barayuda** [comment]
Hi @alexr00 & @gjsjohnmurray 
this screenshot with Bisect Activated
![auto-typing-s-when-switch-workspace_bisect-activated](https://user-images.githubusercontent.com/520463/182111813-deb6c09c-8311-4edd-9ada-fd8e204c1702.gif)

**Comment by gjsjohnmurray** [comment]
@barayuda did you respond to the notification in the lower right? With 0 extensions disabled (the presence of lots of Activity Bar icons is evidence of lots of extensions enabled) you still seem to have the problem. So click the "This is bad" button and follow the next instruction.

**Comment by barayuda** [comment]
@gjsjohnmurray unfortunately, I didn't respond the notification. I will try then 
I will update this asap this issue, thankyou

**Comment by barayuda** [comment]
> @barayuda did you respond to the notification in the lower right? With 0 extensions disabled (the presence of lots of Activity Bar icons is evidence of lots of extensions enabled) you still seem to have the problem. So click the "This is bad" button and follow the next instruction.

@gjsjohnmurray I have click the notification, and give me like this
![image](https://user-images.githubusercontent.com/520463/182284571-7ac8253d-1469-4dae-a10f-e732474043e5.png)

**Comment by alexr00** [comment]
@barayuda which button on the notification did you click?

**Comment by barayuda** [comment]
Hi vscode team and contributor, I would like to say thank you
This issue already fixed, since I upgrade to version 1.70.0 cc @alexr00 @alexdima @gjsjohnmurray 
![image](https://user-images.githubusercontent.com/520463/182993858-2d12c9c0-49a6-42a9-8358-8313901c7826.png)

**preview:**
![preview-on-vscode-1 70 0](https://user-images.githubusercontent.com/520463/182994444-c3de0445-eed7-444c-b9e4-f3d7fea7d40a.gif)
